manim/utils/color.py

```diff
@@ -115,8 +115,9 @@ class Colors(Enum):
 
     .. code-block:: python
 
-       from manim.utils.color import Colors
-       Colors.white.value   # -> '#FFFFFF'
+        >>> import manim.utils.color as C
+        >>> C.WHITE
+        '#FFFFFF'
```

Comment:
Both code-blocks in this file should be doctests, and both should do the same thing as before:

```suggestion
        >>> from manim.utils.color import Colors
        >>> Colors.white.value
        '#FFFFFF'
```